libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1068,25 +1012,32 @@ public String toString() {
     }
 
     /**
-     * Encrypts the given data using the given passcode as the encryption key.
+     * Encrypts the given data.
      *
      * @param data Data to be encrypted.
-     * @param passcode Encryption key.
      * @return Encrypted data.
      */
-    public static String encryptWithPasscode(String data, String passcode) {
-        return Encryptor.encrypt(data, SalesforceSDKManager.INSTANCE.getEncryptionKeyForPasscode(passcode));
+    public static String encrypt(String data) {
+        return Encryptor.encrypt(data, getEncryptionKey());
+    }
+
+    /**
+     * Returns the encryption key being used.
+     *
+     * @return Encryption key.
+     */
+    public static String getEncryptionKey() {
+        return SalesforceKeyGenerator.getEncryptionKey(INTERNAL_ENTROPY);
```

Comment:
getEncryptionKey is called a bunch of times - should we cache the key in memory?